src/components/modals/ExcludeClassTimeFilters.tsx

```diff
@@ -0,0 +1,247 @@
+            const newFilters = filters.filter((it) => it !== filter);
+            storeExcludedClassTimeFilters(newFilters);
+            return newFilters;
+        });
+    };
+
+    const clearFilters = () => {
+        setExcludedClassTimeFilters(() => {
+            const empty: ExcludeClassTimeFilter[] = [];
+            storeExcludedClassTimeFilters(empty);
+            return empty;
+        });
+    };
+
+    return (
+        <Stack
+            direction={"column"}
+            sx={{
+                px: { xs: 0.5, sm: 2 },
+                mt: 1,
+                mx: 3,
+            }}
+            gap={3}
+        >
+            <Stack gap={0.5}>
+                <Typography variant="h6" textAlign={"center"} sx={{ fontSize: 18 }}>
+                    Skjul tidsrom
+                </Typography>
+                <Typography
+                    variant="body2"
+                    textAlign={"center"}
+                    sx={{
+                        color: theme.palette.grey[600],
+                        fontSize: 14,
+                        mb: 1,
+                    }}
+                >
+                    Legg til tidsrom du vil skjule fra timeplanen.
+                </Typography>
+            </Stack>
+            <form
+                onSubmit={(event) => {
+                    event.preventDefault();
+                    addFilter();
+                }}
+            >
+                <FormControl component="fieldset">
+                    <FormGroup sx={{ mt: 1 }}>
+                        <Stack direction={"column"} gap={2}>
+                            <InputLabel id="exclude-class-time-filter-weekday">Ukedag</InputLabel>
+                            <Select
+                                labelId="exclude-class-time-filter-weekday"
+                                id="exclude-class-time-select"
+                                value={excludedClassTimeWeekday}
+                                label="Ukedag"
+                                onChange={({ target: { value } }) => {
+                                    setExcludedClassTimeWeekday(() => value as number);
+                                }}
+                            >
+                                {weekdays.map((day, i) => (
+                                    <MenuItem key={i} value={i + 1}>
+                                        {day}
+                                    </MenuItem>
+                                ))}
+                            </Select>
+                            <Stack direction={"row"} gap={2} alignItems={"center"}>
+                                <TimePicker
+                                    label="Fra"
+                                    views={["hours", "minutes"]}
+                                    ampm={false}
+                                    value={excludedClassTimeStartTime}
+                                    onChange={(newValue) => setExcludedClassTimeStartTime(() => newValue)}
```

Comment:
Any reason for preferring the arrow function?
```suggestion
                                    onChange={(newValue) => setExcludedClassTimeStartTime(newValue)}
```

Reply:
Because I'm not that used to react, and saw it required a `SetStateAction`, unaware that you could simply pass the value :shrug:

Reply:
:white_check_mark: